ui/src/services/hooks/api.ts

```diff
@@ -18,17 +20,30 @@ export const useRunConfig = (): useRunConfigReturn => {
   const ddClient = useDDClient();
   const { data, mutate, isValidating, error } = useSWR(
     cacheKey,
-    () => (ddClient.extension.vm.service.get('/getConfig') as Promise<HTTPMessageBody>),
+    () => (ddClient.extension.vm.service.get('/get') as Promise<HTTPMessageBody>),
```

Comment:
nit: A more canonical way to define this REST API would be to use the HTTP verbs:

```
GET /configs  # get configs
POST /configs  # create a new config
PUT /configs/<id_or_name>  # update an existing config
DELETE /configs/<id_or_name>  # delete an existing config
```

This will make the API a bit easier to evolve, if we start introducing other concepts and resources over time.. 👍

Reply:
I don't know why I thought there were only GET and POST available.....Much better like this :+1: